packages/app-cli/tests/services/plugins/api/JoplinSettings.ts

```diff
@@ -35,6 +35,21 @@ describe('JoplinSettings', () => {
 						label: 'My Custom Setting 2',
 					});
 
+					await joplin.settings.registerSettings({
+						'myCustomSetting3': {
+							value: 3,
+							type: 1,
+							public: true,
+							label: 'My Custom Setting 3',
+						},
+						'myCustomSetting4': {
+							value: 4,
+							type: 1,
+							public: true,
+							label: 'My Custom Setting 4',
+						}
+					})
+
```

Comment:
why did you put that here? Did you check the title of the test, and do you think it's related to your change? So please either find a test related to your changes, or create a new one.

Reply:
This is the only test I found that used `joplin.settings.registerSetting()` and as it is supposed to be replaced by `registerSettings()`, I added this here. I didn't removed the calls to `registerSetting()` though, I'll do that.

Reply:
> why did you put that here? Did you check the title of the test, and do you think it's related to your change? So please either find a test related to your changes, or create a new one.

This is the only test I found that used joplin.settings.registerSetting() and as it is supposed to be replaced by registerSettings(), I added this here. I didn't removed the calls to registerSetting() though, I'll do that.